src/exampleTypes/TagsQuery/TagActionsMenu.js

```diff
@@ -15,7 +15,7 @@ let TagActionsMenu = ({
   return (
     <div
       className="tags-query-tag-actions-menu"
-      style={{ minWidth: 200, padding: 10 }}
+      style={{ minWidth: 100, padding: 10 }}
```

Comment:
Why not just `whitespace: nowrap`?

Reply:
Did you try with it? It did not work for me and that was easier fix since we already have minWidth there etc.

Reply:
No I didn't try it but it would seem like it'd work. Well if it doesn't this is easier